workspaces/quay/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@internal/quay",
+  "version": "1.0.0",
+  "private": true,
+  "engines": {
+    "node": "18 || 20"
+  },
+  "scripts": {
+    "tsc": "tsc",
+    "tsc:full": "tsc --skipLibCheck true --incremental false",
+    "build:all": "backstage-cli repo build --all",
+    "build:api-reports": "yarn build:api-reports:only",
```

Comment:
```suggestion
    "build:api-reports": "yarn build:api-reports:only --tsc",
```

Another odd change, this doesn't match the template: https://github.com/backstage/community-plugins/blob/d41c76d2fc4f8ce1ca937817fe8fd7d222d91fec/workspaces/repo-tools/packages/cli/src/lib/workspaces/templates/workspace/package.json.hbs#L12